src/scripts/contentCapture/pdfScreenshotHelper.ts

```diff
@@ -36,6 +36,7 @@ export class PdfScreenshotHelper {
 
 			let errorCallback = (failureInfo: OneNoteApi.RequestError) => {
 				getBinaryEvent.setStatus(Log.Status.Failed);
+				failureInfo.response = failureInfo.responseHeaders = undefined;
```

Comment:
Just confirming. Here, we intend to set both response and responseHeaders as undefined. Right?

Reply:
Yes.